README.md

```diff
@@ -12,14 +12,12 @@ Get up and running in a few minutes with our drop-in turn-by-turn navigation `Na
 
 ## Features
 
-- Drop-in turn-by-turn navigation UI
-- Automotive, cycling, and walking directions
-- Traffic avoidance
-- Maneuver announcements
-- Text instructions
-- Text to speech via Amazon Polly - no AWS key needed
-- Automatic rerouting
-- Snap to route
+* A full-fledged turn-by-turn navigation UI ready to drop into your application
+* [Professionally designed map styles](https://www.mapbox.com/maps/) for daytime and nighttime driving
+* Worldwide driving, cycling, and walking directions powered by [open data](https://www.mapbox.com/about/open/) and user feedback
+* Traffic avoidance and proactive rerouting based on current conditions in [32 countries](https://www.mapbox.com/api-documentation/pages/traffic-countries.html)
+* Natural-sounding turn instructions powered by [Amazon Polly](https://aws.amazon.com/polly/) (no configuration needed)
+* [Support for 18 languages](./docs/guides/Localization and Internationalization.md)
```

Comment:
This link will still be broken, because the generated docset doesn’t contain a docs/ folder. We could stick an absolute link to the guide here, but it’d be pretty jarring to go from the docset to a Markdown file on GitHub. The correct fix would be to write a proper cover page for the docset instead of reusing the repository’s readme.

Reply:
#1113 adds a cover page. The cover page in that PR doesn’t include a feature list, but now it can without a broken link.